src/Domain/Charity.php

```diff
@@ -57,10 +60,18 @@ class Charity extends SalesforceReadProxy
 
     /**
      * @ORM\Column(type="boolean")
-     * @var bool    Whether the charity's Gift Aid is currently to be claimed by the Big Give.
+     * @var bool    Whether the charity's Gift Aid is expected to be claimed by the Big Give. This
+     *              indicates we should charge a fee and plan to claim, but not that we are necessarily
+     *              set up as an approved Agent yet.
      */
     protected bool $tbgClaimingGiftAid = false;
 
+    /**
+     * @ORM\Column(type="boolean")
+     * @var bool    Whether the charity's Gift Aid may NOW be claimed by the Big Give according to HMRC.
+     */
+    protected bool $tbgApprovedToClaimGiftAid = false;
```

Comment:
Can we do private unless we have a use case for protected?

Separately, I'm wondering whether this should be a required constructor param, so that we know we always have to know whether we're approved or not. We only actually instantiate Charity once in prod code, but it might also be useful to have it as a required constructor param so we're prompted to pass it for tests and we make a choice of which scenario to test instead of just testing the false scenario unless we think of setting it to true.